test/rest.browser.test.js

```diff
@@ -30,7 +30,7 @@ describe('strong-remoting-rest', function() {
 
   // setup
   beforeEach(function() {
-    objects = RemoteObjects.create();
+    objects = RemoteObjects.create({cors: false});
```

Comment:
Ah! I didn't think about those messages crowding the test results!

I think you should leave them in.